kvbc/include/sparse_merkle/internal_node.h

```diff
@@ -166,6 +166,7 @@ std::ostream& operator<<(std::ostream& s, std::optional<T> const& v) {
 class BatchedInternalNode {
  public:
   static constexpr size_t MAX_CHILDREN = 31;
+  static constexpr size_t MAX_HEIGHT = 5;
```

Comment:
why so ?

Reply:
The height in a single record (sub-tree) is 5, because we have a binary tree and 16 elements in the the bottom of the record. It is shown in the following documentation: [internal_node.h:114](https://github.com/vmware/concord-bft/blob/66b249a1e709ed4d1baf5f4e5e73bae19bbadd58/kvbc/include/sparse_merkle/internal_node.h#L114)

Reply:
It would be good to put a comment to explain this.

Reply:
Even better would be to add a constexpr function which deduces this number from the parameters (max key length, etc...)

Reply:
Yes, will add the constexpr function, I think it will be better to calculate the `MAX_HEIGHT` based on the `MAX_CHILDREN` considering that out MPT is binary.